sdk/core/Azure.Core/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.2.0-preview.1 (Unreleased)
 
+- Add `AzureKeyCredential` and its respective policy.
```

Comment:
Should be under an `### Added` heading according to [guidelines](https://azure.github.io/azure-sdk/policies_releases.html#changelog-guidance).